genesyscloud/resource_genesyscloud_journey_segment_test.go

```diff
@@ -16,12 +15,8 @@ import (
 	"github.com/mypurecloud/platform-client-sdk-go/v130/platformclientv2"
 )
 
-func TestAccResourceJourneySegmentCustomer(t *testing.T) {
-	runResourceJourneySegmentTestCase(t, "basic_customer_attributes")
```

Comment:
This test was supposed to be related to customer scope. There's no scope anymore, so this was doing the same as the test below.

Got rid of it and left just one in place.